src/index.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
+
 import './index.css';
 import App from './App';
+
```

Comment:
use import correctly, re order this and group the imports according to what they do, for example: react dependenncies, compontents/modules/, and css at the end